assets/js/components/HealthSummary/HomeHealthSummary.jsx

```diff
@@ -52,7 +52,10 @@ const healthSummaryTableConfig = {
       render: (content, item) => {
         const linkToCluster = `/clusters/${item.clusterId}`;
```

Comment:
Hey can we use an if statement to check if there is an clusterId? 
If there is none, we don't render the Link. 
Otherwise, we just hide the rendered Link.

What do you think ?

Reply:
As discussed, that would mean having something like
```javascript
if (!item?.clusterId) {
  # here the HealthIcon needs to be instructed about the hover opacity
  return <HealthIcon health={content} centered={true} hoverOpacity={'0'}/>;
}

const linkToCluster = `/clusters/${item.clusterId}`;
return (
  <Link to={linkToCluster}>
    <HealthIcon health={content} centered={true} />
  </Link>
);
```
which I'd be fine with, yet it would require making an extra change to the `HealthIcon` component to accept the opacity, because it defaults to `hover:opacity-75` while in a specific scenario has to be 0.

I find enough disabling `pointerEvents` when we do not have a clusterId, keeping the scope narrow.
Still open to other's stance on this.

Reply:
> I find enough disabling pointerEvents when we do not have a clusterId

We don't change how event propagation works for a corner case. We change the rendered DOM instead. That's the point about React.

@EMaksy is right, we'll have to modify the `HealthIcon` using a prop accordingly, maybe a boolean one. Also, you might find useful what React docs say about conditional rendering :smile: 

https://reactjs.org/docs/conditional-rendering.html

Reply:
right, already two in that direction. Changing it!

Reply:
Actually let me rephrase: changing because you are right. 

It semantically does not make sense having a Link which is actually not going to be a Link.
I overlooked that semantics, my bad :sweat_smile: 

Thanks guys! :heart: